### ISSUE TITLE
HTTP 500: Internal Server Error - No such profile : Jupyterhub on GKE

### ISSUE THREAD

**Issue by SalmaAmrAhmed** [GoogleCloudPlatform/ai-notebooks-extended#12]
I have followed the guidelines exactly and it was working fine for a while, but then I tried to login the URL and I got this error when I selected the deployed image then clicked "start".
I cleaned everything and tried to install it again and that's when I reproduced the error. 

<img width="1792" alt="Screen Shot 2021-06-15 at 4 15 17 PM" src="https://user-images.githubusercontent.com/6736993/122068764-f7eab300-cdf4-11eb-8ec6-630634f0d162.png">

I dug more into the container logs and it shows the following:

```
[I 2021-06-15 11:34:09.056 JupyterHub log:181] 302 GET /hub/ -> /hub/spawn (salma.abdelfattah@talabat.com@::ffff:10.80.1.5) 12.73ms
[I 2021-06-15 11:34:09.298 JupyterHub log:181] 200 GET /hub/spawn (salma.abdelfattah@talabat.com@::ffff:10.80.1.5) 4.13ms
[E 2021-06-15 11:34:11.988 JupyterHub user:690] Unhandled error starting salma.abdelfattah@talabat.com's server: No such profile: ['gcr-io-tlb-data-prod-jupyter-mine-basic']. Options include: gcr-io-tlb-data-prod-jupyter-mine-basic
[E 2021-06-15 11:34:12.021 JupyterHub pages:261] Failed to spawn single-user server with form
    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/jupyterhub/handlers/pages.py", line 257, in post
        return await self._wrap_spawn_single_user(
      File "/usr/local/lib/python3.8/dist-packages/jupyterhub/handlers/pages.py", line 314, in _wrap_spawn_single_user
        raise web.HTTPError(
    tornado.web.HTTPError: HTTP 500: Internal Server Error (Error in Authenticator.pre_spawn_start: ValueError No such profile: ['gcr-io-tlb-data-prod-jupyter-mine-basic']. Options include: gcr-io-tlb-data-prod-jupyter-mine-basic)

[I 2021-06-15 11:34:12.023 JupyterHub log:181] 200 POST /hub/spawn (salma.abdelfattah@talabat.com@::ffff:10.80.1.5) 91.20ms
```

Any idea how can I fix this?

**Comment by m-mayran** [comment]
This seems to be related to kubespawner 1.0.0. Still investigating. But for now, I set the Kubespawner version to 0.16.1 (the previous one)

Should be fixed with the latest commit [c537965](https://github.com/GoogleCloudPlatform/ai-notebooks-extended/commit/c5379650c6f73c3cb4398e3477425b6800ed3651)

**Comment by m-mayran** [comment]
Closing this as it seems to have fixed the issue for now.
